modules/extension/xsd/xsd-core/pom.xml

```diff
@@ -70,17 +70,6 @@
     </developer>
   </developers>
 
-
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>commons-collections</groupId>
-        <artifactId>commons-collections</artifactId>
-        <version>3.2.2</version>  <!-- Updated for security (CVE-2015-8103) binary compatible with 3.1 -->
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
-
   <!-- =========================================================== -->
```

Comment:
this dependency management version 3.2.2 is already covered in root pom.xml (also we do not use dependencyManagement in child pom.xml files right?)